### PR TITLE
Update node tutorial docs

### PR DESCRIPTION
**Pull request by simlay**
I mostly just updated some of the start steps and made sure the whole tutorial worked.

[Rendered](https://github.com/simlay/fluvio-website/blob/cloud-launch-updates/content/tutorials/node/simple-chat.md) without images of course.

### CODE REVIEW

File: content/tutorials/node/simple-chat.md [hunk 1]
```diff
@@ -53,16 +53,25 @@ From the `chat-app` directory, run `npm run build`. This will run the `./build.s
 
 ### **Run the Server**
 
-Open a new terminal window, navigate to the `chat-server` directory and start the application:
+Open a new terminal window, navigate to the `chat-server` directory, setup the fluvio topics and start the application:
 
 ```bash
```
Comment (ajhunyady) [inline content/tutorials/node/simple-chat.md:58]: for consistency, we should use the same 1 liner as in the client (&& to chain commands)

Comment (ajhunyady) [reply content/tutorials/node/simple-chat.md:58]: cd chat-server && npm run setup && npm run start

**Comment by ajhunyady** [comment]
@simlay could you please also update the screenshot? It looks better with multiple messages form different users.

**Comment by simlay** [comment]
> @simlay could you please also update the screenshot? It looks better with multiple messages form different users.

[Done.](https://github.com/infinyon/fluvio-website/blob/3cb42b3d82a9d4a292164705a3396d5a082ef7b4/static/img/tutorials/chat-app/chat.png)

**Pull request merged**